### DISCUSSION TITLE
Animated Drawings in MAUI .NET?

### DISCUSSION THREAD

**Discussion by pvoelker** [dotnet/maui#12707] [Q&A] [answered]
I'm working on a game for kids.  I want to implemented animated drawings when someone gets an answer rights.

I know that MAUI .NET supposedly supports animated GIFs.  However they have such a limited color palette.  I was hoping there would be something a little more robust.

Is there any support for APNGs (animated PNGs) or other alternative ways to display animated drawings?

**Comment by xtuzy** [comment, marked as answer]
- you can draw a image as a frame, see https://learn.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/bitmaps/animating
- skiasharp can show lottie animation
